src/mesh/parallel/shiftedmetric.cxx

```diff
@@ -136,18 +136,41 @@ void ShiftedMetric::cachePhases() {
  * and Y is then field aligned.
  */
 const Field3D ShiftedMetric::toFieldAligned(const Field3D& f, const REGION region) {
-  return shiftZ(f, toAlignedPhs, region);
+  switch (f.getDirectionY()) {
+  case (DIRECTION::YOrthogonal):
+    return shiftZ(f, toAlignedPhs, DIRECTION::YOrthogonal, region);
```

Comment:
This should use `DIRECTION::YAligned`

Reply:
Good spot! Guess this shows I should add more unit tests...